Tests/AlicerceTests/Network/Pinning/Certificates/CertificateGenerationSteps.md

```diff
@@ -0,0 +1,98 @@
+# Create OpenSSL Certificates
```

Comment:
Agree, even though this is only useful if one wants to re-generate certificates in the future, which shouldn't happen since we "force" a constant certificate verify date when we create our test `SecTrust`'s 😉. 

I thought it would be nice to leave that info on the repo nonetheless because it took me quite some time to gather and get it right 😆, and might come in handy in the future.

Reply:
Indeed. It was a good call. 👏